index.js

```diff
@@ -8,10 +8,11 @@ var appName = require(path.join(process.cwd(), 'package.json')).name;
 var eyes = new Eyes();
 
 var eyesOpen = false;
-var hooked = browser.get;
+var browserGetSnapshotEnabled = false;
```

Comment:
1. default should be `true`
2. please name the `before` and `after` by the same name convention, I prefer a configuration object holding both

Reply:
About (2): `beforeSnapshotEnabled` is a miss-leading name, if a test case has more then one `browser.get` then it will get 2 snapshots for each `browser.get`, thus a correct name is `browserGetSnapshotEnabled`.

What did you mean by " I prefer a configuration object holding both"?